src/components/Projects.tsx

```diff
@@ -1,7 +1,149 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/knife.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+import Background from "./assets/home-page/background.jpg";
+import {
+  color,
+  space,
+  typography,
+  TypographyProps,
+  ColorProps,
+  SpaceProps,
+} from "styled-system";
+
+type HeadingProps = SpaceProps & ColorProps & TypographyProps;
+
+const Header = styled.div<HeadingProps>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+`;
+
+const Title = styled.h1<HeadingProps>`
+  ${space}
+  ${color}
+  ${typography}
+  font-family: SangBleu OG Serif Light Regular;
+`;
+
+const IconContainer = styled.div`
+  width: 200px;
+  height: 120px;
+`;
+
+const Icon = styled.a`
```

Comment:
I think `a` is inappropriate here because these aren't links. Maybe `span`?

Reply:
Even if they will link to each corresponding project page?

Reply:
Sorry, no, I hadn't considered that they will be links to pages!! You're absolutely right (though we'll likely use `Link` from `react-router`, but let's cross that bridge when...)